internal/controller/controllers/agentserviceconfig_controller_test.go

```diff
@@ -1656,7 +1656,7 @@ var _ = Describe("getOSImages", func() {
 	})
 })
 
-var _ = Describe("Default ConfigMap values", func() {
+var _ = Describe("Assisted Service ConfigMap", func() {
```

Comment:
This is really testing `newAssistedCM` right?

If so, that's what should be in this `Describe`.

Reply:
Yes, thank you! Updated to reflect this